deploy/charts/harvester/values.yaml

```diff
@@ -400,3 +400,22 @@ longhorn:
     concurrentAutomaticEngineUpgradePerNodeLimit: 3
 
 rancherEmbedded: false
+
+webhook:
+  replicas: 1
```

Comment:
It should work. Just note that [resource locking like this](https://github.com/harvester/harvester/blob/8cfc827fa20a3b5c9bdb357e02369bd61dea3ba9/pkg/api/user/store.go#L57-L70) doesn't work (I'm not sure if it works in current code either since we have multiple harvester backend instances).

Each instance does its resource protection doesn't make too much sense. I'd suggest doing this kind of checks in the leader controller.

Reply:
Sounds reasonable to me

Reply:
[Make default value to 3. ](https://github.com/harvester/harvester/pull/919/commits/40599eaeb4538db4eda0c97b305053378dadd143#diff-1e557ee14cc96a67bfde9c7a2f9b5099c4453a72d8cbe98dd3babfce02a1557bR404-R406)

Reply:
We also need anti-affinity rules so that all replicas don't run on the first node. e.g., https://github.com/harvester/harvester/blob/master/deploy/charts/harvester/templates/deployment.yaml#L32

Reply:
Addressed in this [commit].(https://github.com/harvester/harvester/pull/919/commits/c10ad67a0cdd4dc30c54ef84d8604ddd5547cb53)